feconf.py

```diff
@@ -669,7 +669,7 @@ def get_empty_ratings():
 LIBRARY_SEARCH_URL = '/search/find'
 LIBRARY_SEARCH_DATA_URL = '/searchhandler/data'
 LIBRARY_TOP_RATED_URL = '/library/top_rated'
-MERGE_SKILL_URL = '/merge_skill'
+MERGE_SKILLS_URL = '/merge_skill'
```

Comment:
Can we use the URL /merge_skills_handler or /mergeskillshandler?

Reply:
Done